pypeit/images/sciencecube.py

```diff
@@ -0,0 +1,256 @@
+""" Object to hold + process a single image.
+This module also includes the build_from_list() method
+which is how the ScienceImage is most frequently generated. """
+
+import inspect
+
+import os
+import numpy as np
+from shapely.geometry import Polygon
+
+from pypeit import msgs
+
+from pypeit.core import procimg
+from pypeit.par import pypeitpar
+from pypeit import utils
+
+from pypeit.images import pypeitimage
+from pypeit.images import combineimage
+
+from IPython import embed
+
+
+class ScienceImage(pypeitimage.PypeItImage):
```

Comment:
Name `ScienceCube`

Reply:
I've renamed this, but I'm not sure I put a lot of effort into this code... seems I created it to put in the overlapping pixel/voxel code mainly... Deleting from this PR, and will bring it forward to the next PR.